recipes-core/systemd/systemd/imx-generic-bsp/systemd-poweroff.service

```diff
@@ -0,0 +1,20 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=System Power Off
+Documentation=man:systemd-poweroff.service(8)
+DefaultDependencies=no
+Requires=shutdown.target umount.target final.target
+After=shutdown.target umount.target final.target
+# change to systemctl --force halt instead of shutdown-force this to avoid pmic power off on hmx since the watchdog driver will send a shutdown message instead.
```

Comment:
```suggestion
# Using `systemctl --force halt` instead of `shutdown-force` is to avoid PMIC power off on HMX. The watchdog driver will trigger shutdown from the M core instead.
```